public/app/plugins/datasource/grafana-testdata-datasource/tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "../../../../../tools/plugins/shared/tsconfig.json",
```

Comment:
It might be worth trying to turn the shared plugin configs into a private yarn workspace. I believe all the tooling could then "extend" it just by doing `@grafana/plugin-config/tsconfig.json` etc.

Reply:
seems an overkill to share a json file? What would be the benefits of doing that?

Reply:
I was thinking more about extending any of the files in the current `tools/plugins/shared` location not just this tsconfig.json file. Just think it makes the paths less bothersome, can move the configs around with minimum effort. 🤷